egs/madcat_ar/v1/local/chain/run_cnn_chainali_1b.sh

```diff
@@ -0,0 +1,226 @@
+#!/bin/bash
+
```

Comment:
@aarora8, there should be results at the top of these files, obtained from the compare_wer.sh script.

Reply:
sorry, I realized I haven't run run_cnn_1a.sh, run_cnn_chainali_1b and run_cnn_end2endali_1a, with the recent code. I have currently ran run_flatstart_cnn1a.sh and run_cnn_end2endali_1b (UC) scripts. I updated the results on those two scripts, I will run and update the results for other scripts aswell.